webpack/assets/javascripts/react_app/common/hooks/API/fixtrues.js

```diff
@@ -0,0 +1,19 @@
+export const resultsFromLOTR = {
```

Comment:
Looks like we need to disable spelling linting for this file. Also, there's a typo in the file name - should be `fixtures.js`. Actually, if you call it `API.fixtures.js` then eslint ignore will skip it (https://github.com/theforeman/foreman/blob/develop/.eslintignore#L7)